jenkins-scripts/lib/colcon-default-devel-windows.bat

```diff
@@ -84,6 +83,12 @@ echo # BEGIN SECTION: configure the MSVC compiler
 call %win_lib% :configure_msvc2019_compiler
 echo # END SECTION
 
+:: Prepare a clean vcpkg environment with external dependencies
+call %win_lib% :remove_vcpkg_installation || goto :error
+echo # BEGIN SECTION: install all vcpkg dependencies
+call %win_lib% :setup_vcpkg_all_dependencies || goto :error
+echo # END SECTION

```

Comment:
It would be nice (maybe a follow-up PR) to print the list of installed packages and their versions for debugging purposes.

Reply:
+100. I'll do it after merging this PR.

Reply:
https://github.com/gazebo-tooling/release-tools/pull/908